spec/services/api_client_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "rails_helper"
+
+RSpec.describe ApiClient do # are these covered by the gem tests?
+  subject { described_class }
+
+  describe "class methods" do
+    it "calls the defined methods" do
+      expect(subject).to receive(:get_teaching_subjects)
+      subject.get_teaching_subjects
+
+      expect(subject).to receive(:get_candidate_initial_teacher_training_years)
+      subject.get_candidate_initial_teacher_training_years
+
+      expect(subject).to receive(:get_qualification_degree_status)
+      subject.get_qualification_degree_status
+
+      expect(subject).to receive(:get_qualification_uk_degree_grades)
+      subject.get_qualification_uk_degree_grades
+
+      expect(subject).to receive(:get_qualification_types)
+      subject.get_qualification_types
+
+      expect(subject).to receive(:get_country_types)
+      subject.get_country_types
+
+      expect(subject).to receive(:get_candidate_retake_gcse_status)
+      subject.get_candidate_retake_gcse_status
+
+      expect(subject).to receive(:get_callback_booking_quotas)
+      subject.get_callback_booking_quotas
+    end
+  end
+
+  describe "ping test" do # just a healthcheck for the dev api
```

Comment:
Should this be left in?

Reply:
Just a reminder if we need a ping test. It works on local, but fails on build, happy to get rid, probably covered by Steve somewhere....